app/controllers/user.js

```diff
@@ -97,6 +97,9 @@ function updateUserById(req, res, next) {
 	let update = Object.assign({id: req.params.id}, req.body);
 	User.forge(update)
 		.save(null, {method: 'update'})
+		.then(function(model) {
+			return model ? model.fetch() : model;
```

Comment:
Couple things here:
1. The update promise resolves with an object with only the updated fields in it (what you're calling `model` on line 100). In JavaScript, that will always evaluate as truthy, therefore you're always returning whatever `model.fetch()` returns.
2. `model.fetch()` won't do anything when called like this. You need to specify what you're trying to fetch. You also can't call this on the updated object the previous promise resolves with. You need to call it on `User` itself.
3. The return for the `else` case is not a promise, so the following `then()` block will never be called
4. Smaller nitpick, but you're using the old style function syntax. Instead of `function(model) {...` you should use `model => {...`